server/core/backend.cc

```diff
@@ -223,6 +223,7 @@ bool Backend::connect(MXS_SESSION* session, SessionCommandList* sescmd)
 bool Backend::write(GWBUF* buffer, response_type type)
 {
     mxb_assert(in_use());
+    m_backend->lru_clock = mxs_lru_clock();
```

Comment:
The way Backend uses it is different from how readconnroute uses it. Backend sets the clock on each write whereas readconnroute sets it on each connection. Since these are shared between different routers, they should behave similarly.